Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+
+MRP Specific Rules
+
+[color=#a4885c]14.[/color] This server is a Roleplay Expected server. We define this as performing your assigned duties. This is a ‘do your job’ server. This means it is important to do what is expected out of your department and not what would be expected out of other departments.
+    - Don’t do other peoples jobs for them. Opt in for the role you intend to play or change your job IC. Failure to do your basic duties may result in a job ban.
+
+[color=#a4885c]15.[/color] Each round exists independently of all other rounds. While friendships persisting through rounds is okay, using experiences from other rounds to give an advantage or disadvantage to other players is not permitted.
+
+[color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living.
+
+Note: SSD characters are players who are disconnected or AFK. It is possible for players to return to SSD bodies. Players become catatonic when they take a ghost spawner role or commit suicide. Players are NOT able to return to these bodies without admin intervention.
+
+[color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable, but killing, looting, or otherwise griefing them while they are away is not.
+    - If the player in question is an antag’s target or they are being secured by security, interactions to finish what is required of your duties/objectives are always acceptable.
+    - Ahelping about an SSD/AFK player may grant an exception to this rule.
+    - This does not apply to players that are catatonic. Although needlessly killing or harming catatonic players is discouraged.
+
+[color=#a4885c]18.[/color] We now adhere to our own Standard Operating Procedure and Alert Procedure policy. These can be found at wiki.spacestation14.io.
+        Calling the shuttle to end the round faster despite there being no significant threats to the station will likely result in it being recalled.
+
+Command Rules
+
+[color=#a4885c]19.[/color] The following jobs: Captain, HOP, HOS, CE, RD, CMO, and AI should ahelp when they need to leave.
+    - These roles must exhibit some competency. Lack of this can result in a job ban. Command roles are designed to lead and manage departments first and do second. They are not intended to become do-it-all members of their departments.
+    - Crew promoted to acting heads of staff must step down when an official head of staff arrives at the station. This is to prevent confusion with the chain of command when the new player arrives.
+
+[color=#a4885c]20.[/color] In the absence of a captain, the Head of Personnel or Head of Security must serve as acting captain or split the duties of the captain between themselves. In the absence of a captain, HoP, and HoS, the other command staff have equal standing in determining the acting captain.
```

Comment:
Just make it explicitly the HoP unless there is no HoP, it'll make things more straightforward for everyone.